grpc/java/BUILD

```diff
@@ -38,6 +38,14 @@ java_grpc_compile(
     ]
 )
 
+java_grpc_compile(
+    name = "protocol-cluster-src",
+    deps = [
+        "//protobuf/cluster:cluster-proto",
+        "//protobuf/cluster:discovery-proto"
+    ]
+)
+
```

Comment:
We could, maybe, split up the folders differently, and create `protobuf/core:answer-proto` (etc.) and `protobuf/cluster:cluster-proto` - what do you think?

I actually kind of prefer it the way it is now, but just throwing out an idea

Reply:
I'm also not sure yet.

I think it's best to keep the change minimal and keep the current structure, because we're going to see more protocol definitions added for other cluster features such as backup and authentication. That would be the point where we can think about how to re-optimise the structure.

What do you think @alexjpwalker ?

Reply:
Yeah I agree with that.